lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -3094,6 +3094,8 @@ Name | Description
 ``\${GALAXY_SLOTS:-4}`` | Number of cores/threads allocated by the job runner or resource manager to the tool for the given job (here 4 is the default number of threads to use if running via custom runner that does not configure GALAXY_SLOTS or in an older Galaxy runtime).
 ``\$GALAXY_MEMORY_MB`` | Total amount of memory in megabytes (1024^2 bytes) allocated by the administrator (via the resource manager) to the tool for the given job. If unset, tools should not attempt to limit memory usage.
 ``\$GALAXY_MEMORY_MB_PER_SLOT`` | Amount of memory per slot in megabytes (1024^2 bytes) allocated by the administrator (via the resource manager) to the tool for the given job. If unset, tools should not attempt to limit memory usage.
+``\$_GALAXY_JOB_HOME_DIR`` | Path to an empty directory in the job's woring directory that can be used as a home directory.
+``\$_GALAXY_JOB_TMP_DIR`` | Path to an empty directory in the job's woring directory that can be used as a temporary directory.
```

Comment:
```suggestion
``\$_GALAXY_JOB_HOME_DIR`` | Path to an empty directory in the job's working directory that can be used as a home directory.
``\$_GALAXY_JOB_TMP_DIR`` | Path to an empty directory in the job's working directory that can be used as a temporary directory.
```

Actually I think it's better to not document `$_GALAXY_JOB_HOME_DIR`, it shouldn't be needed in tools.

Reply:
I have seen tools using `$HOME` to write config files. Which is particularly annoying in real user setups. In case `$HOME` is not set (question above) then I would like to leave it.